core/bin/external_node/src/main.rs

```diff
@@ -206,7 +206,7 @@ async fn run_core(
 
     let (persistence, miniblock_sealer) = StateKeeperPersistence::new(
         connection_pool.clone(),
-        config.remote.l2_erc20_bridge_addr,
+        config.remote.l2_shared_bridge_addr.unwrap(),
```

Comment:
consider using 'expect' rather than unwrap - it makes debugging a lot easier.

(especially if this is some config related thing - we can clearly give the message what field is missing)